docs/reference/ccr/index.asciidoc

```diff
@@ -3,27 +3,168 @@
+
+NOTE: You cannot manually modify the mapping or alias of a follower index.
+
+If you apply a non-dynamic settings change to the leader index that is
+needed by the follower index, the follower index closes itself, applies the
+settings update, and then re-opens itself. The follower index is unavailable
+for reads and cannot replicate writes during this cycle.
+
+[discrete]
+[[ccr-remote-recovery]]
+=== Remote recovery process
+When you create a follower index, you cannot use it until it is fully
+initialized. The _remote recovery_ process builds a new copy of a shard on a
+follower node by copying data from the primary shard in the leader cluster.
+
+{es} uses this remote recovery process to bootstrap a follower index using the
+data from the leader index. This process provides the follower with a copy of
+the current state of the leader index, even if a complete history of changes
+is not available on the leader due to Lucene segment merging.
+
+Remote recovery is a network intensive process that transfers all of the Lucene
+segment files from the leader cluster to the follower cluster. The follower
+requests that a recovery session be initiated on the primary shard in the
+leader cluster. The follower then requests file chunks concurrently from the
+leader. By default, the process concurrently requests five 1MB file
+chunks. This default behavior is designed to support leader and follower
+clusters with high network latency between them.
+
+TIP: You can modify dynamic <<ccr-recovery-settings,remote recovery settings>>
+to rate-limit the transmitted data and manage the resources consumed by remote
+recoveries.
+
+Use the <<cat-recovery,recovery API>> on the follower cluster to obtain
+information about an in-progress remote recovery. Because {es} implements
+remote recoveries using the <<snapshot-restore,snapshot and restore>>
+infrastructure, running remote recoveries are labelled as type
+`snapshot` in the recovery API.
+
+[discrete]
+[[ccr-leader-requirements]]
+=== Requirements for leader indices
+{ccr-cap} works by replaying the history of individual write
+operations that were performed on the shards of the leader index. The history
+of these operations needs to be retained on the leader shards
+so that they can be pulled by the follower shard tasks. The underlying
+mechanism used to retain these operations is _soft deletes_.
+
+A soft delete occurs whenever an existing document is deleted or updated. By
+retaining these soft deletes up to configurable limits, the history of
+operations can be retained on the leader shards and made available to the
+follower shard tasks as it replays the history of operations.
+
+Soft deletes must be enabled for indices that you want to use as leader
+indices. Soft deletes are enabled by default on new indices created on
+or after {es} 7.0.0.
+
+// tag::ccr-existing-indices-tag[]
+IMPORTANT: {ccr-cap} cannot be used on existing indices. If you have
```

Comment:
This feels like a point-in-time statement that's a little misleading now--would be good to verify. Can you set up CCR for an existing index that does have soft deletes enabled?

Reply:
Verified with @dnhatn that this statement should read:
>IMPORTANT: Cross-cluster replication cannot be used on existing indices created using Elasticsearch
7.0.0 or earlier, where soft deletes are disabled. You must reindex your data into a new index with soft deletes enabled.